### ISSUE TITLE
[Feature Request] Option to move tracklist album's title allignment to the left

### ISSUE THREAD

**Issue by glu8716** [Taiko2k/TauonMusicBox#999]
I'd love to have an option to achieve something like this:

![2](https://github.com/Taiko2k/TauonMusicBox/assets/33501455/9da14c5e-8347-4efb-9475-cf7690b6f617)

Move the album's title to the left while keeping the release date on the right.
Of course this is not something that has to be mandatory but just an option to have more customization.

**Comment by Solarunit** [comment]
When I first starting to use tauon it was a bit weird to have album's titles on the right. Nowadays I am fine with it, it helps to distinguish tracks from the album's titles. I think this option could be there, but not sure about "release date on the right" , for me it's duration of the album in the line  - the last item.

**Comment by glu8716** [comment]
You are right, there is also the album duration, I forgot to include that, but my main point is to have the album's title on the left aligned with tracklist. Both album duration and date could be on the right, or one could decide what to put on the left and what to put on the right. This would be a great improvement for user customization.

**Comment by Taiko2k** [comment]
A setting for this is doable. A fully customized system is probably more than I want to get into right now, but may become necessary in the future if too many of these settings get stacked on.

Anyway here's a prototype:

![Screenshot from 2023-06-28 23-20-46](https://github.com/Taiko2k/TauonMusicBox/assets/17271572/26a7e2a7-c2b5-4ca3-8ffa-41cc873599c0)

Duration would be added to the right of the date if enabled.

Only issue is the alignment of the title, in your mock-up you had it aligned with the track artist, but im inclined to leave it more aligned to the left such as in the image. What do you think? 

(Thing is its a different size of font, so won't fully align the full length of the text anyway)

**Comment by glu8716** [comment]
I'd be okay with your solution too, but I'd personally prefer to have it aligned with the artist, it looks prettier to me.
What about adding two separate font size editors? One for the album's title, duration and date and one for the tracklist?
One could then make the two fonts the same size and this wouldn't cause a misalignment.

**Comment by Solarunit** [comment]
>but im inclined to leave it more aligned to the left such as in the image.

I think I like this alignment. 

I am not sure if I like that some album info on the right side, I would like to have everything on the left side (when this option is enabled)

**Comment by StrangeSeasOfThought** [comment]
That's one of the first things I modified back in the day. ^^ I didn't like and didn't need to see the same artist name in every single line of the album.

Moved the artist name to the top, left aligned, together with the album title and used a more vibrant/brighter color. Genres to the right, followed by release year, in a less vibrant/slightliy darker color.

![tracklist](https://github.com/Taiko2k/TauonMusicBox/assets/43582772/4872321f-4ab9-4ecb-aac8-e9ce93f39bca)

**Comment by Taiko2k** [comment]
@StrangeSeasOfThought That actually looks pretty nice. I can see the value of customising this now.

I think the best way forward is if I make some sort of special json config file, flexible enough so the end-user can achieve these without needing to edit the source code. Ill update with further progress.

**Comment by StrangeSeasOfThought** [comment]
> I think the best way forward is if I make some sort of special json config file, flexible enough so the end-user can achieve these without needing to edit the source code.

Sounds good.

**Comment by Solarunit** [comment]
>I think the best way forward is if I make some sort of special json config file, flexible enough so the end-user can achieve these without needing to edit the source code. Ill update with further progress.

I am thinking about something like visual editor, template, several predefined templates with a preview maybe?

**Comment by StrangeSeasOfThought** [comment]
@Taiko2k I noticed that the new separator option overwrites any "-" in album titles when I don't select "-" as the separator.
